web/containers/admin/AssessmentPanel/index.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { firebaseConnect, firestoreConnect, isLoaded } from 'react-redux-firebase';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import AssessmentPanel from '../../../components/admin/AssessmentPanel';
+
+class AssessmentPanelContainer extends React.Component {
+  static contextTypes = {
+    store: PropTypes.object.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      // one of 'hacker', 'mentor', 'volunteer'
+      applicantType: 'hacker',
+      // all documents from <applicantType>_main_info collection
+      applicants: {},
+      // single document from <applicantType>_main_info for selected applicant
+      selectedApplicantMainInfo: {},
+      // single document from <applicantType>_long_info collection for selected applicant
+      selectedApplicantLongInfo: {},
+    };
+  }
+
+  componentWillMount() {
+    this.setState({ applicants: this.getApplicants() });
```

Comment:
should we also get the long info of the first applicant here by default?

Reply:
Thanks for pointing this out, fixed the issue by just showing the `ApplicantList` (width 100%) and then showing the score panel and applicant info panel only if some applicant is selected.